### ISSUE TITLE
[MU4 Issue] No obvious way to create a double-dotted note

### ISSUE THREAD

**Issue by wizofaus** [musescore/MuseScore#12333]
**Describe the bug**
In v3 there are two simple ways to create a double-dotted note - one via the note input toolbar, and secondly via the "ctrl+." shortcut. Neither work in v4.

**To Reproduce**
1. Create a new score in 4/4 and add a half-note
2. Select it and hit Ctrl+. to convert it to a double-dotted half-note.
3. It remains as an undotted half note with no obvious alternative method to change to double-dotted.

**Expected behavior**
Ctrl+. should work as per v3 and ideally there's away to do so e.g. by long-pressing on the "augmentation dot" icon in the note input toolbar.

**Screenshots**

**Platform information**
Windows 10

**Additional context**
Running locally built from latest source

**Comment by wizofaus** [comment]
So apparently it is possible, by clicking the "gear" icon in the note input toolbar than clunkily scrolling through the huge list of options to find "double augmentation dot" option.  I had looked at that gear icon but didn't even realise the list of was scrollable.

**Comment by MarcSabatella** [comment]
Just the double-dot button to your toolbar via the customize facility (gear icon).

Not sure what you were expected Ctrl+. to do; that's not a built-in shortcut in any version of MuseScore that I am aware of.

**Comment by wizofaus** [comment]
I don't remember setting that Ctrl+. shortcut myself in v3, but it's definitely one I have there.  At any rate v4 doesn't even allow you to create such a shortcut.
Double-dotted notes are rare enough that I wouldn't want a toolbar icon dedicated to them, but there still needs to be a way to create as needed.

**Comment by Tantacrul** [comment]
@cbjeukendrup / @shoogle - will the double augmentation dot be revealed when we add the list of remaining actions to the shortcuts panel? I would hope that at least provides a way to assign a shortcut for it. At that point, it may be decided to assign a key by default too.

- Apart from that, we intentionally left the double dot option as a feature you have to switch on in the Note Input bar (using the cog). In my opinion, this is an option that kind of sits in limbo. It _is_ important but I also suspect not _quite_ enough to warrant being on by default. I could be persuaded either way, depending on user reaction. It's one we'll gauge once we begin testing in Beta.

**Comment by wizofaus** [comment]
I'd still think some usability testing on the ability to reveal dditional buttons on the note input toolbar is warranted - it really wasn't obvious to me at all,  and I'm usually pretty persistent trying to find features.  I honestly thought it had been accidentally overlooked.

**Comment by cbjeukendrup** [comment]
@Tantacrul Yes, after fixing the list of shortcuts, the double dot action will appear in there too. 

For the rest: personally I agree that it's fine that double dot doesn't have a default shortcut (just like MU3) and also doesn't appear in the default toolbar. Adding it to the toolbar seems easy enough to me, except that users who are used to having scrollbars always visible might not notice that the list of available actions is scrollable because MuseScore doesn't show scrollbars. If that turns out to confuse more people, we should consider changing that, but that's a separate issue anyway.

**Comment by wizofaus** [comment]
The "gear" icon still doesn't seem an obvious place to look for a "double dot" command though (vs., as per my suggestion, long press or right click on the single dot command).

**Comment by Tantacrul** [comment]
OK, so I'm closing this issue, since the double-dot is going to be added to shortcuts in a day or two anyway. I'm not taking on any other suggestions. If there are UX issues, we'll discover them in proper user testing.
